src/Client/AzureManaged/Client.AzureManaged.csproj

```diff
@@ -0,0 +1,24 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <PackageDescription>Azure Managed extensions for the Durable Task Framework client.</PackageDescription>
+    <EnableStyleCop>true</EnableStyleCop>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="../../Client/Grpc/Client.Grpc.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Azure.Identity" Version="1.13.1" />
+    <PackageReference Include="Microsoft.Extensions.Options.DataAnnotations" Version="8.0.0" />
```

Comment:
We are targeting net6 but using a v8.x package. While this is technically 'valid', it may be confusing to users. Are we able to use a 6.x package here?

Reply:
![{791DA31B-A16A-4053-8E00-379D46984313}](https://github.com/user-attachments/assets/02cde451-7738-4bf3-bc3b-7f659c6c10a6) v6 of this package does not target dotnet 6, and v7 does not have 
![{16E38BDC-D45C-4567-ADB7-9851B06452D0}](https://github.com/user-attachments/assets/364f63d3-bf43-469b-ba56-0008c86546a8) validateonstart. v8 is min version targeting dotnet 6 and has validateonstart api

Reply:
Do you need ValidateOnStart? This will effectively be validated on start, as validation will occur on first import of it, which will be done at start due to this being eventually imported by a `IHostedService` (the DurableTaskWorker)

Reply:
good point, updated ver and removed validateonstart